src/site/xhtml/installation.xhtml.vm

```diff
@@ -9,33 +9,11 @@
 
 	<h1>ICAT Clients Installation</h1>
 
-
-
-	<p>Note that the setup script only installs the Python client as
-		there is no standard place to install jar files.</p>
-
```

Comment:
I'd suggest to add a note that icat.client does not need to be installed at all.  Something like:

> icat.client only consists of a jar file that is made available in the maven repository of the ICAT project. It does not need to be installed locally. In order to use it in a Java application, add a dependency on the appropriate version of icat.client to the pom file of your application. The build process will then fetch it from the repository as needed.

Reply:
I added a note based on your suggestion.